tools/LLVMOptimizer.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include "llvm/IR/Function.h"
+#include "llvm/IR/LLVMContext.h"
+#include "llvm/IR/LegacyPassManager.h"
+#include "llvm/IR/Module.h"
+#include "llvm/IR/Verifier.h"
+#include "llvm/Passes/PassBuilder.h"
+#include "llvm/Support/Error.h"
+
+#include <string>
+
+
+llvm::Module *optimize_module(std::string optArgs,llvm::Module *M);
```

Comment:
this should take const char* as input to avoid making string copies.

Reply:
I used llvm::StringRef for comparison convenience. Do you think it is acceptable?